docs/conf.py

```diff
@@ -31,6 +31,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    'sphinx.ext.doctest',
```

Comment:
Ah I had I assumed `make html` ran the doctests when the plugin was there. Cool.